packages/ui/src/components/form.tsx

```diff
@@ -71,7 +71,7 @@ Form.Field = function FormField({
 
 type InputFieldProps = FieldProps<string> &
   Pick<FormFieldProps, 'description' | 'label' | 'required'> &
-  Pick<InputProps, 'disabled' | 'placeholder'>;
+  Pick<InputProps, 'disabled' | 'placeholder' | 'type'>;
```

Comment:
I don't think we need to add the changes here for 'type'. We're already verifying that only numbers can be inputted in `student.ts`, and doing this adds an unnecessary icon on the right side of the input field. Lmk if there was any other reason you think we need this 

<img width="688" alt="Screenshot 2024-09-01 at 11 09 25 AM" src="https://github.com/user-attachments/assets/252db639-7ead-40fe-8cff-4b841e954e5c">